<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EVAN License Key Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="7" column="1" >
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Generate Key</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>MAC Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>License Key</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QTextEdit" name="textEdit" />
    </item>
    <item row="3" column="1" >
     <widget class="QTextEdit" name="textEdit_2" />
    </item>
    <item row="6" column="1" >
     <widget class="QDateEdit" name="dateEdit" />
    </item>
    <item row="4" column="1" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Expiration Date</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
